mono/metadata/marshal.c

```diff
@@ -7809,8 +7811,8 @@ mono_marshal_get_native_wrapper (MonoMethod *method, gboolean check_exceptions,
 			ret->param_count = csig->param_count + 1;
 			ret->ret = csig->ret;
 			for (int i = 0; i < csig->param_count; ++i) {
-				// FIXME: TODO implement handle wrapping for out and inout params.
-				g_assert (!mono_signature_param_is_out (csig, i));
+				if (mono_signature_param_is_out (csig, i))
```

Comment:
Shouldn't the check be for outarg and ref type? 
Otherwise we'll handle'ize `static extern void Foo(out int bar)`.